components/Form/index.js

```diff
@@ -75,6 +77,8 @@ export default function Form({ onSubmit, onCancel, project = {} }) {
     projectData.instructions = steps;
     projectData.tools = tools;
     projectData.material = material;
+    projectData.author = session.user.email;
```

Comment:
usually, you would like to add this information to the data in the backend (with getServerSession). A hacker could change this information here. Critical information like the user data should always be handled in a spot where it cannot be changed by a user / hacker.